### ISSUE TITLE
Remote control apps

### ISSUE THREAD

**Issue by fulldecent** [calabash/calabash-ios#1252]
Is there a use case to remotely control apps in realtime (not automated) from a Mac?

Basically you plug in the phone, compile/recompile/iresign an existing app, then access that app using your Mac with touch events being played on the phone.

This would be helpful for many use cases other than developing apps.

**Comment by jmoody** [comment]
I am afraid I don't understand.  Can you rephrase this request?

**Comment by fulldecent** [comment]
Is it possible to use calabash as a remote USB touchscreen to control an iPhone?

**Comment by jmoody** [comment]
Are you asking, "Can I use Calabash iOS to perform gestures on an iPhone connected by USB to my Mac?"

If so, then yes.  

If not, I still don't understand what you are asking.

**Comment by fulldecent** [comment]
Sorry, let me try again.

Can I perform gestures on the iPhone in real-time just like a mouse/trackpad provides realtime interaction with a computer? Or instead, must all gestures be created ahead of time and then played back all at once?

**Comment by jmoody** [comment]
I think you want the Calabash iOS console.

```
# Target a simulator
$ bundle exec calabash-ios console
> start_test_server_in_background
```

**Comment by fulldecent** [comment]
Beautiful, thank you for explaining.
